plugins/BEdita/Core/src/Model/Behavior/DataDefaultsBehavior.php

```diff
@@ -51,6 +53,11 @@ class DataCleanupBehavior extends Behavior
     public function beforeMarshal(Event $event, \ArrayObject $data)
     {
         $config = $this->getConfig();
+        $objectType = strtolower($this->_table->getAlias());
```

Comment:
```suggestion
        $objectType = Inflector::underscore($this->_table->getAlias());
```
`MyTypes` should become `my_types` not `mytypes`

Reply:
:+1: applyed in latest commit